app/assets/javascripts/main.js.erb

```diff
@@ -271,6 +271,27 @@ $(function() {
       }
     }
   }
+  /* Ajax call to reopen the 'right' info window for the current state */
+  function refreshInfoWindow() {
+    if (activeThingId == null 
+      || activeThingId === null 
+      || activeThingId === undefined) {
```

Comment:
I'm so bad at JavaScript that I always rely on lodash for these types of things.

https://lodash.com/docs/4.17.11#isNil

Reply:
Is lodash dependent on nodeJS?

Reply:
nodejs is installed in docker but I don't see it in AAD.

Reply:
lodash is a front end JS library with no dependencies.